kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -1,263 +1,386 @@
+	sudo cp $AZURESTACK_ROOT_CERTIFICATE_SOURCE_PATH $AZURESTACK_ROOT_CERTIFICATE_DEST_PATH
+	
+    AZURESTACK_ROOT_CERTIFICATE_SOURCE_FINGERPRINT=`openssl x509 -in $AZURESTACK_ROOT_CERTIFICATE_SOURCE_PATH -noout -fingerprint`	
     AZURESTACK_ROOT_CERTIFICATE_DEST_FINGERPRINT=`openssl x509 -in $AZURESTACK_ROOT_CERTIFICATE_DEST_PATH -noout -fingerprint`
 
-    echo "AZURESTACK_ROOT_CERTIFICATE_SOURCE_FINGERPRINT: $AZURESTACK_ROOT_CERTIFICATE_SOURCE_FINGERPRINT"
-
-    echo "AZURESTACK_ROOT_CERTIFICATE_DEST_FINGERPRINT: $AZURESTACK_ROOT_CERTIFICATE_DEST_FINGERPRINT"
+    log_output -i "AZURESTACK_ROOT_CERTIFICATE_SOURCE_FINGERPRINT: $AZURESTACK_ROOT_CERTIFICATE_SOURCE_FINGERPRINT"
+    log_output -i "AZURESTACK_ROOT_CERTIFICATE_DEST_FINGERPRINT: $AZURESTACK_ROOT_CERTIFICATE_DEST_FINGERPRINT"
 
     update-ca-certificates
+
+	# Azure CLI specific changes.
+	export REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
+    sudo sed -i -e "\$aREQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt" /etc/environment
+	
     AZURESTACK_RESOURCE_METADATA_ENDPOINT="$TENANT_ENDPOINT/metadata/endpoints?api-version=2015-01-01"
     curl $AZURESTACK_RESOURCE_METADATA_ENDPOINT
 }
 
-echo "Update the system."
+# Basic details of the system
+log_output -i "Starting Kubernetes cluster deployment."
+log_output -i "Running  script as : $(whoami)"
+
+log_output -i "System information: $(sudo uname -a)"
+WAIT_TIME_SECONDS=20
+log_output -i "Waiting for $WAIT_TIME_SECONDS seconds for system to get into stabe state."
+sleep $WAIT_TIME_SECONDS
+
+# Script parameters
+log_output -i "------------------------------------------------------------------------"
+log_output -i "Input parameter details:"
+log_output -i "ADMIN_USERNAME: $ADMIN_USERNAME"
+log_output -i "AGENT_COUNT: $AGENT_COUNT"
+log_output -i "AGENT_SIZE: $AGENT_SIZE"
+log_output -i "IDENTITY_SYSTEM: $IDENTITY_SYSTEM" 
+log_output -i "K8S_AZURE_CLOUDPROVIDER_VERSION: $K8S_AZURE_CLOUDPROVIDER_VERSION"
+log_output -i "MASTER_COUNT: $MASTER_COUNT"
+log_output -i "MASTER_DNS_PREFIX: $MASTER_DNS_PREFIX"
+log_output -i "MASTER_SIZE: $MASTER_SIZE"
+log_output -i "PUBLICIP_DNS: $PUBLICIP_DNS"
+log_output -i "PUBLICIP_FQDN: $PUBLICIP_FQDN"
+log_output -i "REGION_NAME: $REGION_NAME"
+log_output -i "RESOURCE_GROUP_NAME: $RESOURCE_GROUP_NAME"
+log_output -i "SSH_PUBLICKEY: $SSH_PUBLICKEY"
+log_output -i "STORAGE_PROFILE: $STORAGE_PROFILE"
+log_output -i "TENANT_ID: $TENANT_ID"
+log_output -i "TENANT_SUBSCRIPTION_ID: $TENANT_SUBSCRIPTION_ID"
+log_output -i "------------------------------------------------------------------------"
+
+#####################################################################################
+# Install all prequisite. 
+log_output -i "Update the system to latest."
 retrycmd_if_failure 5 10 sudo apt-get update -y
 
-echo "Installing pax for string manipulation."
+log_output -i "Installing pax for string manipulation."
```

Comment:
We may not need the pax package, I do not see any usages.

Reply:
We will remove in future checkin.